src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/mapper/RoleAssignmentToAccessProfileMapperImpl.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.mapper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.AccessProfile;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.GrantType;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignment;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignmentFilteringResult;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseTypeDefinition;
+import uk.gov.hmcts.ccd.domain.model.definition.RoleToAccessProfileDefinition;
+
+@Component
+public class RoleAssignmentToAccessProfileMapperImpl implements RoleAssignmentToAccessProfileMapper {
+
+    @Override
+    public List<AccessProfile> toAccessProfiles(RoleAssignmentFilteringResult filteringResult,
+                                                CaseTypeDefinition caseTypeDefinition) {
+        List<RoleAssignment> roleAssignments = extractRoleAssignments(filteringResult);
+
+        if (hasGrantTypeExcluded(roleAssignments)) {
+            roleAssignments = filterRoleAssignments(roleAssignments);
+        }
+
+        List<AccessProfile> accessProfiles = new ArrayList<>();
+        for (RoleAssignment roleAssignment : roleAssignments) {
+            for (RoleToAccessProfileDefinition roleToAccessProfile : caseTypeDefinition.getRoleToAccessProfiles()) {
+                if (validRoleAssignments(roleToAccessProfile, roleAssignment)) {
+
+                    List<String> authorisationsList = Arrays.asList(roleToAccessProfile.getAuthorisation().split(","));
```

Comment:
```suggestion
                    List<String> authorisations = Arrays.asList(roleToAccessProfile.getAuthorisations());
```
i.e. add a method to AccessProfile that does the split